app/styles/layout/calendar.css

```diff
@@ -0,0 +1,72 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
I don't think you need to import that. We don't import it in other files. That is managed by remix vite plug-in

Reply:
Resolved!